doc/sphinx/addendum/ring.rst

```diff
@@ -102,15 +102,15 @@ forget this paragraph and use the tactic according to your intuition.
 Concrete usage in Coq
 --------------------------
 
-.. tacn:: ring
+.. tacv:: ring
```

Comment:
Moving text between 2 chapters we've not converted is out of scope for this PR.  Nonetheless, I moved the description of `ring` into the ring chapter.  I made the `field*` tactics point to that chapter as well.

For the ones in the SSR chapter, each of them has its own `tacname (ssreflect)` variant defined in the chapter, so I think they should indeed be tacvs.

Reply:
My comment did not apply to the SSR chapter. AFAICT the SSR chapter just repeats everything at the bottom, so this could result in the need to use `tacv` indeed.